raw_data/preprocessed/africa/benin_french.txt

```diff
@@ -0,0 +1,303 @@
+
+(4) Les principes généraux figurant à l'article 3 ci-dessus visent les objectifs suivants: [a] protéger l'environnement, notamment: [•] prévenir et anticiper les actions de nature à avoir des effets immédiats ou futurs sur la qualité de l'environnement; [•] faire cesser toute pollution ou dégradation, ou tout au moins en limiter les effets négatifs sur l'environnement; [•] promouvoir l'assainissement dans le but d'améliorer le cadre de vie; [•] surveiller étroitement et en permanence la qualité de l'environnement; [b] restaurer les zones et sites dégradés; [c] assurer l'équilibre entre l'environnement et le développement.
+
+(5) Pour atteindre les objectifs prévus à l'article 4, des dispositions sont prises en vue de: [a] élaborer et exécuter un programme national de développement des capacités en environnement; [b] effectuer des recherches sur la qualité de l'environnement au sein d'organismes publics ou privés; [c] promouvoir l'information et l'éducation relatives à l'environnement par les organismes publics et privés; [d] établir les normes de la qualité de l'environnement ainsi que celles du rejet; [e] établir et gérer un système d'information permanent sur la qualité de l'environnement, en particulier sur les éléments naturels et les industries à risque; [f] élaborer et mettre en oeuvre une politique nationale d'aménagement du territoire. Aux fins ci-dessus, le gouvernement doit: [•] produire un rapport annuel sur l'état de l'environnement au Bénin; [•] publier les données statistiques disponibles relativement à la qualité de l'environnement; [•] acquérir, construire et implanter sur tout point du territoire du Bénin tous équipements nécessaires à la surveillance de la qualité de l'environnement et, à ces fins, instituer toute servitude et acquérir tout immeuble nécessaire par tous moyens légaux; [•] obtenir tout renseignement nécessaire à l'application de la loi; [•] conclure dans l'intérêt de la République du Bénin et en conformité avec les lois, et règlements en vigueur, tout accord avec tout autre gouvernement ou organisme international afin de faciliter l'exécution de la présente loi; [•] faciliter la création et le fonctionnement d'associations de protection, de défense et de mise en valeur de l'environnement, tant au niveau national que local; ces organismes peuvent être associés aux actions entreprises par le gouvernement, notamment en matière d'information, d'éducation et de communication des citoyens et être reconnus d'utilité publique; [•] rechercher systématiquement la consultation ainsi que le niveau d'intervention le plus efficace pour la mise en oeuvre de la présente loi, conformément à la politique nationale de déconcentration et de décentralisation. 
```

Comment:
'; ces organismes' is '. Ces organismes' in the legislation. Not sure if this is an intentional change.

Reply:
I modified the original text intentionally. It seemed inappropriate to interrupt the list partway through with a period.